scripts/check_cron_registry.py

```
@@ -0,0 +1,268 @@
+    if not rows:
+        raise ValueError(f"No expected cron rows parsed from {path}")
+    return rows
+
+
+def load_live_jobs(path: Path) -> Dict[str, Dict[str, Any]]:
+    data = json.loads(path.read_text(encoding="utf-8"))
+    jobs = data.get("jobs") if isinstance(data, dict) else data
+    if isinstance(jobs, dict):
+        jobs = list(jobs.values())
+    if not isinstance(jobs, list):
+        raise ValueError(f"Unsupported jobs.json structure in {path}")
```

Ruff (0.15.12)

[warning] Prefer `TypeError` exception for invalid type

(TRY004)